napari/_vispy/vendored/volume.py

```diff
@@ -530,8 +530,8 @@
 
 
 class VolumeVisual(Visual):
```

Comment:
Can you removed the changed to this vendored file. It will likely cause merge conflicts with some other PRs in progress and they are only formatting changes